examples/package.json

```diff
@@ -9,10 +9,13 @@
     "dev": "vite",
     "build": "vite build",
     "preview": "vite preview",
-    "test:all": "echo 'No tests available' && exit 0"
+    "test:all": "playwright test"
   },
   "devDependencies": {
+    "@playwright/test": "^1.52.0",
```

Comment:
Please install `@playwright/test` on the top level (i will need it in `lib` also soon) and remove unnecessary deps. `@types/node` already exist on top level and `cypress` is unused.